k8s/tools/cli/loadtest/cronjob.yaml

```diff
@@ -0,0 +1,58 @@
+---
+#
+# Copyright (C) 2019-2020 Vdaas.org Vald team ( kpango, rinx, kmrmt )
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: vald-loadtest
+  labels:
+    app: vald-loadtest
+    app.kubernetes.io/name: vald
+    app.kubernetes.io/component: loadtest
+spec:
+  schedule: "*/1 * * * *"
```

Comment:
Is this okay to run every 1 minute?

Reply:
its temporary user should change this param with configmap